home.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The LOTR Lovers Conference</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <div>
+        <div class="popUp flex">
+            <ul id="popUpItems" class="popUpItems flex">
+               <a href="#about">
+                 <li class="popUpItem">
+                About
+                </li>
+            </a>
```

Comment:
- [ ] Kindly note that it is a requirement that users be able to navigate between the home to the about page with ease. To implement this, you need to lin the path to the about.html page in the href attribute. You could do so like this:

```
               <a href="./about.html">
                 <li class="popUpItem">
                About
                </li>
            </a>
```